docs/data-sources/gateway.md

```diff
@@ -0,0 +1,236 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "encore_gateway Data Source - terraform-provider-encore"
+subcategory: ""
+description: |-
+  Encore provisioned gateway information
+---
+
+# encore_gateway (Data Source)
+
+Encore provisioned gateway information
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the Encore resource
+
+### Optional
+
+- `env` (String) The environment of the Encore resource. Defaults to the provider environment
+
+### Read-Only
+
+- `aws_alb` (Attributes) (see [below for nested schema](#nestedatt--aws_alb))
+- `aws_fargate_task_definition` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition))
+- `cluster_ip` (Attributes) (see [below for nested schema](#nestedatt--cluster_ip))
+- `gcp_cloud_run` (Attributes) (see [below for nested schema](#nestedatt--gcp_cloud_run))
+- `k8s_container` (Attributes) (see [below for nested schema](#nestedatt--k8s_container))
+- `k8s_ingress` (Attributes) (see [below for nested schema](#nestedatt--k8s_ingress))
```

Comment:
It's not exactly clear which of these are set and in what situations

Reply:
yeah, I haven't added docs yet. Just the structure :)